.github/workflows/main.yml

```diff
@@ -0,0 +1,63 @@
+name: Run tests
+
+on:
+  [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: Ubuntu-20.04
+
+    env:
+      DB_TYPE: postgresql
+      DB_USERNAME: postgres
+      DB_PASSWORD: postgres
+      DB_ENDPOINT: localhost:5432 
+      DB_NAME: bitschema
+      DB_TEST_NAME: bit_schema_test
+      
+    services:
+      postgres:
+        image: postgres
+        env: 
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+    steps:
+    - uses: actions/checkout@v2
+    - name: Connect to PostgreSQL
+      run: |
+        PGPASSWORD=postgres psql -h localhost -c 'CREATE DATABASE bit_schema_test;' -U postgres
+        PGPASSWORD=postgres psql -h localhost -c 'CREATE SCHEMA bitschema;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c 'CREATE SCHEMA test_schema;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c 'create SCHEMA test_schema_2;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c '\dn;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c 'show search_path;' -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c "ALTER DATABASE bit_schema_test SET search_path TO bitschema,public;" -U postgres -d bit_schema_test
+        PGPASSWORD=postgres psql -h localhost -c 'show search_path;' -U postgres -d bit_schema_test
+      env:
+        POSTGRES_HOST: localhost
+        POSTGRES_PORT: 5432
+
+    - uses: actions/checkout@v2
```

Comment:
is this duplication from line 27? I don't think we need it again here, do we?

Reply:
@epicadk , @vj-codes , do we need this line (43)

Reply:
Yes, it is not needed again.